### DISCUSSION TITLE
Add support for bracket object selectors for token function arguments.

### DISCUSSION THREAD

**Discussion by soroush96** [chakra-ui/panda#1961] [Ideas]
### Description

we can't use negative values of the tokens we have set. for example `token(sizes.-xl)` or `token(sizes["-xl"])` wouldn't work.

### Problem Statement/Justification

to be able to use some of the non conventional object keys it is better to have bracket selector in use in tokens

### Proposed Solution or API

mentioned above

**Comment by astahmer** [comment]
you can do it like this https://play.panda-css.com/uMLFKDjxw7
